src/server/_utils/pricing.ts

```
@@ -0,0 +1,52 @@
+  USER_ROLES.FOUNDING_MEMBER,
+  USER_ROLES.FOUNDER,
+  USER_ROLES.AMBASSADOR,
+];
+
+export function getPricingForRole(role: string): PricingInfo {
+  const isFounding = FOUNDING_ROLES.includes(role as UserRole);
+
+  if (isFounding) {
+    return {
+      monthlyPrice: 10,
+      annualPrice: 99,
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'annualPrice' should be before 'monthlyPrice'.

(sort-keys-fix/sort-keys-fix)